src/Bicep.Core.Samples/Files/Resources_CRLF/main.bicep

```diff
@@ -265,4 +265,47 @@ resource existing2 'Mock.Rp/existingExtensionResource@2020-01-01' existing = {
 resource extension3 'My.Rp/extensionResource@2020-12-01' = {
   name: 'extension3'
   scope: existing1
+}
+
+param shouldDeployChildAndGrandChild bool = true
+resource nestedA 'My.Rp/nestedResource@2020-01-01' existing = {
+  name: 'nestedA'
+
+  resource nestedB 'childType' = if (shouldDeployChildAndGrandChild) {

```

Comment:
>nestedB [](start = 11, length = 7)

Do we show the fully qualified or relative/partial type when you hover over a tested resource or its symbol elsewhere?

Reply:
Will be testing this out today and report back.

I think we discussed the hover showing the fully-qualified name during the design meeting.

Reply:
Hover displays the fully-qualified name (without any changes on my part).